### ISSUE TITLE
Files output with nonsense text

### ISSUE THREAD

**Issue by kanetik** [ayvazj/gradle-androlate-plugin#1]
I ran the task and got a lot of text that looks like this:

  <string name="activate">??????</string>
  <string name="deactivate">???????</string>
  <string name="premium_upgrade_success">??????????????????????????????</string>
  <string name="help">???</string>
  <string name="add">??</string>
  <string name="upgrade_to_premium">??????????????</string>
  <string name="why">???</string>

I had previously tried Android Localizationer, and that had pretty much the same issue: https://github.com/westlinkin/AndroidLocalizationer/issues/17 - it was an encoding issue.

**Comment by ayvazj** [comment]
Can you provide a simple strings.xml that reproduces the issue and include the androlate block from your build.gradle file.

**Comment by kanetik** [comment]
Didn't even notice it had stripped out the tags from around the ???...  Here's the English:

```
<resources>
    <string name="deactivate">Deactivate</string>
    <string name="bulb_description">Wakey Toggle</string>
    <string name="premium_upgrade_success">Thank you for upgrading to premium!</string>
    <string name="help">Help</string>
    <string name="add">Add</string>
    <string name="upgrade_to_premium">Upgrade to Premium!</string>
    <string name="why">Why?</string>
</resources>
```

and the Japanese:

```
<resources>
    <string name="deactivate">??????</string>
    <string name="bulb_description">???????</string>
    <string name="premium_upgrade_success">??????????????????????????????</string>
    <string name="help">???</string>
    <string name="add">??</string>
    <string name="upgrade_to_premium">??????????????</string>
    <string name="why">???</string>
</resources>
```

And finally, the Androlate block (minus the API Key andproject name):

```
androlate {
    appName '{project name was here, but not gonna include that here}'
    apiKey '{key was here, but not gonna include that here}'
    defaultLanguage 'en'
    targetLanguages = ['ja']
}
```

**Comment by ayvazj** [comment]
I wasn't able to reproduce the issue on OS X or Windows 10 but I'm assuming your default encoding is not UTF-8.  I published version 0.2 that explicitly sets the encoding of the output file to UTF-8 if you would like to try it.

```
classpath('com.github.ayvazj.gradle.plugins.androlate:androlate:0.2')
```
